content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -423,54 +425,20 @@ model Category {
 }
 ```
 
-### Relation tables
-
-A relation table (also sometimes called _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modeling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
-
-When using Prisma, you can create relation tables by defining [models](/concepts/components/prisma-schema/data-model#defining-models) similar to how you would model them as tables. The main difference is that the fields of the relation table are both annotated relation fields with a corresponding relation scalar field.
-
-Relation tables are also often used to add "meta-information" to a relation. For example, to store _when_ the relation was created.
-
-Here is an example for a relation table called `CategoriesOnPosts`:
-
-```prisma
-model Post {
-  id         Int                 @id @default(autoincrement())
-  title      String
-  categories CategoriesOnPosts[]
-}
+## Relation tables
```

Comment:
Why this change in headline level? Now it is on the same level as "Relational databases" and "MongoDB", which does not seem correct.

Reply:
The intention was to separate it out as a concept to act as a glossary item. Rather than a different type of relation strategy.

Reply:
This page defines how relations work in Prisma on either relational databases, or MongoDB. I don't think "Relational tables" (which are only relevant for relational databases) can live on the same level as these.

Reply:
Hmm, I agree. I'll move it to h3 again, the content should be clear enough

Reply:
fixed in latest commit